.github/workflows/release_crypto_js.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow releases the `matrix-sdk-crypto-js` project.
+#
+# It is triggered when a new tag matches
+# `matrix-sdk-crypto-js-v[0-9]+.*`, which then build the package for
+# the binding, run its tests to ensure everything is still correct,
+# and publish the package on NPM.
+
+
+name: Release `crypto-js`
+
+env:
+  CARGO_TERM_COLOR: always
+  PKG_PATH: "bindings/matrix-sdk-crypto-js"
+
+on:
+  push:
+    tags:
+      - matrix-sdk-crypto-js-v[0-9]+.*
+
+jobs:
+  publish-matrix-sdk-crypto-js:
+    name: Publish 🕸 [m]-crypto-js
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+          profile: minimal
+          override: true
+
+      - name: Load cache
+        uses: Swatinem/rust-cache@v1
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.0
+
+      - name: Install NPM dependencies
+        working-directory: ${{ env.PKG_PATH }}
+        run: npm install
+
+      - name: Configure NPM auth token
+        working-directory: ${{ env.PKG_PATH }}
+        run: npm set "//registry.npmjs.org/:_authToken" "${{ secrets.NPM_TOKEN }}"
```

Comment:
that looks peculiar, is that really the official way to do this?

Reply:
I've explained this in this comment: https://github.com/matrix-org/matrix-rust-sdk/pull/966#issuecomment-1222151714

Reply:
right ... you explain why that means `npm login` isn't necessary, but not why we want to skip that in the first place ... googling a bit I understand now that `npm login` doesn't support `token`s natively (WHAT?!?) and that it would require some unnecessary http requests. so this sort of is the best way to _just set the token_ as ugly as this is.

Reply:
> I understand now that npm login doesn't support tokens natively (WHAT?!?)

Yeah, I had the exact same reaction…

> this sort of is the best way to just set the token as ugly as this is.

Exact!